code/Gamemodes/Gamemode.cs

```diff
@@ -5,15 +5,13 @@ public partial class Gamemode : Entity
 {
 	public virtual string GamemodeName => "";
 
-	[Net]
-	public int PlayerCount { get; private set; }
-
 	/// <summary>
-	/// All players in the game, including dead players.
-	/// Excludes disconnected players.
+	/// A list of currently connected players that are actively apart of the gamemode.
 	/// </summary>
 	[Net]
-	public IList<Player> PlayerList { get; set; }
+	public IList<Player> Players { get; set; }
+
+	public List<Player> DisconnectedPlayers { get; set; } = new();

```

Comment:
Do we care about privating this?

Reply:
Go for it. We can always make it public later if we feel like we want it exposed.